Source/WarpX.H

```diff
@@ -1279,7 +1280,7 @@ private:
 #if (defined WARPX_DIM_RZ) && (defined WARPX_USE_PSATD)
     amrex::Vector<std::unique_ptr<PML_RZ> > pml_rz;
 #endif
-
+    amrex::Real v_particle_pml;
```

Comment:
Let's add an empty line below `v_particle_pml` here, since the variables declared below that one (`moving_window_x` and `current_injection_position` do not belong do the group of PML variables):
```suggestion
    amrex::Real v_particle_pml;
    
```